plugins/lookup/passwordstore.py

```diff
@@ -178,12 +208,17 @@ def parse_params(self, term):
                         self.paramvals[key] = util.strtobool(self.paramvals[key])
             except (ValueError, AssertionError) as e:
                 raise AnsibleError(e)
+            if self.paramvals['missing'] not in ['error', 'warn', 'create', 'empty']:
+                raise AnsibleError("{0} is not a valid option for missing".format(self.paramvals['missing']))
             if not isinstance(self.paramvals['length'], int):
                 if self.paramvals['length'].isdigit():
                     self.paramvals['length'] = int(self.paramvals['length'])
                 else:
                     raise AnsibleError("{0} is not a correct value for length".format(self.paramvals['length']))
 
+            if self.paramvals['create']:
+                self.paramvals['missing'] = 'create'
```

Comment:
@aminvakil because of the above `paramvals['create']` does not need to be checked.